### DISCUSSION TITLE
GRAIN BOUNDARY CHARACTERISTIC DISTRIBUTION

### DISCUSSION THREAD

**Discussion by mugikumar** [mtex-toolbox/mtex#1802] [Ask Anything]
Hi developers,
Five parameter grain boundary distribution- three of them are lattice misorientation, and two of them are boundary plane orientation. For lattice misorientation can quickly get from conventional EBSD, boundary plane orientation needs 3D EBSD. G.S Rohrer et al. suggested with the help of a stereology approach analysing the boundary plane distribution means found either tilt or twist boundaries. In mtex documentation also mentioned is possible to get the trace for tilt boundaries. I need a code for the pole figure with grain boundary distribution for different plane normal means I am analyzing how changes crystallographic plane for intervariant boundaries (in specimen cordinates). the input data are variant boundaries are misoriented by 60 degree and rotated by direction of [11-1] Euler angle must be defined by grain boundaries how axes are misoriented by 60 degree. could you please help me to define those parameter in the MTex code (misorintation angle and axis)
gb = [grains.boundary('indexed','indexed') grains('indexed').innerBoundary]
mori = gb.misorientation
mdf = calcDensity(mori,'contourf')
axSpecimen = ebsd.orientations .* mdf.axis('noSymmetry'); (here my doubt is how to define the particular misorientation with corresponding axis)

plot(axSpecimen,'contourf','fundamentalRegion','antipodal','halfwidth',2.5*degree)
mtexColorbar('title','distribution of misorientation axes in mrd')
![MTEX DOUBT](https://user-images.githubusercontent.com/109027461/234181285-041d8dfa-573f-41e6-9618-d2c15bc13139.jpg)

**Comment by ralfHielscher** [comment]
Does this https://mtex-toolbox.github.io/BoundaryNormalDistribution.html documentation page help?

Ralf.

**Comment by mugikumar** [comment]
Hi Ralf,
 Thanks for your reply, i couldn't see distribution documentation before but i checked it. There is no stereographic projection code in that documentation.

**Reply by ralfHielscher** [reply]
The "stereographic projection code" is inside the function ```calcGBPD```.

**Reply by mugikumar** [reply]
Hi Ralf,
      Sorry to disturb you again. I have done a stereographic projection for 2D EBSD. i am only choosing the misorientation angle between the grains. what i was getting from code i am upload down and what i need if same misorientation with different rotation axis how to do in mtex gbpd code. 
Also i need greater circle in stereographic projection (code used 
figure;plot(gbnd1,'contourf')
mtexColorbar 
hold on
cs  = crystalSymmetry('m-3m')
plot(cs)
hold off this one is coming with specimen symmetry quads, square like that)
 i have upload tsl map of same misorientation angle with different axis direction
![stereo graphic projection](https://user-images.githubusercontent.com/109027461/234650492-47b60467-70ed-437f-9cde-d9a8106d8a4e.jpg)
![10 53 0-1-1](https://user-images.githubusercontent.com/109027461/234652605-aecc6c0d-ccfb-43d9-a557-3248fa932dc7.jpg)
![10 53 11-1](https://user-images.githubusercontent.com/109027461/234652614-94cdd55a-d89c-4fd7-9936-44f492513972.jpg)

**Reply by nyyssont** [reply]
I don't think there is support in MTEX right now for what you want to do. Currently symmetry is taken into account when boundary habit planes are calculated.

**Reply by mugikumar** [reply]
Thanks, nyyssont for your reply. Is this possible to get the code  of the greater circle include in the stereographic projection.

**Comment by mugikumar** [comment]
Hi ralf 
Thanks for sharing the code. You are creating the code with the help of the stereological method. All bicrystals have a common axis with the same direction in the reference frame of both crystals. Rotations about this axis, [uvw], take the two crystals from the coincident configuration at ω = 0 to misoriented configurations of ω about [uvw]. So, for any axis-angle combination, distribution of grain boundary planes can be plotted as illustrated. So he concludes with traces of tilt and twist could you please provide the code of white line traces what they showed in image
![mtex st doubt](https://user-images.githubusercontent.com/109027461/234851771-00529e53-fc11-4a25-8328-f46f30989031.jpg)
